.pylintrc

```diff
@@ -15,7 +15,8 @@ disable=
     too-many-statements,
     ungrouped-imports,
     wrong-import-order,
-    unspecified-encoding
+    unspecified-encoding,
+    consider-using-f-string
```

Comment:
https://github.com/aiven/pglookout/pull/75

Reply:
Thank you, was too tired yesterday evening to go thought the cases...